pkg/kubectl/kubectl.go

```diff
@@ -66,7 +57,7 @@ func parseVersionLine(line string) (string, error) {
 	return "", nil
 }
 
-func parseVersionOutput(stdout string, versionData *kubeCtlVersionData) (err error) {
+func parseVersionOutput(stdout string) (clientVersion, serverVersion string, err error) {
```

Comment:
What is the reason behind naming return values, as you don't seem to use them as return values?